docs/installation/uninstall.md

```diff
@@ -0,0 +1,105 @@
+---
+title: Uninstall Opni
+slug: /installation/uninstall
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Before uninstalling all capabilities should be removed from all clusters.  This can be done by clicking an installed capability in the clusters screen and choosing to remove the cabability.
+
+<Tabs>
+<TabItem value="rancher" label="Uninstall in Rancher UI" default>
+
+**Due the presence of a Kubernetes finalizer the Logging backend should be disabled before Opni is uninstalled in the central cluster.  This can be done in the Opni Management UI by going to the Logging backends in the left hand navigation pane, and selecting Disable.**
```

Comment:
```suggestion
**Due to the presence of a Kubernetes finalizer, the Logging backend should be disabled before Opni is uninstalled in the central cluster.  This can be done in the Opni Management UI by going to the Logging backends in the left hand navigation pane, and selecting Disable.**
```